Tests/Unit/Model/PaypalPaymentMethodTest.php

```diff
@@ -33,14 +33,14 @@ public function testGetConfig()
          */
         $oPayment = PaymentMethodHelper::getPaymentById(PaypalPaymentMethod::getName(true));
 
-        $oConfig = $this->oPaymentMethod->getConfig($oPayment);
+        $oConfig = $this->_oPaymentMethod->getConfig($oPayment);
         $this->assertNotNull($oConfig);
         $this->assertNotNull($oConfig->get('paypal'));
     }
 
     public function testGetTransaction()
     {
-        $oTransaction = $this->oPaymentMethod->getTransaction();
-        $this->assertInstanceOf(PayPalTransaction::class, $oTransaction);
+        $oTransaction = $this->_oPaymentMethod->getTransaction();
+        $this->assertTrue($oTransaction instanceof \Wirecard\PaymentSdk\Transaction\PayPalTransaction);
```

Comment:
Please import `\Wirecard\PaymentSdk\Transaction\PayPalTransaction` and only use `PayPalTransaction` to make it more readable.

Reply:
👍